resources/validations/ui/src/context/shared/saxon-js-gateway.ts

```diff
@@ -60,3 +62,217 @@ export const SaxonJsSchematronValidatorGateway =
+                        baseUri:
+                          'file:///Users/dan/src/10x/fedramp-automation/resources/validations/ui/identity.xsl',
+                        slots: '200',
+                        line: '3',
+                        module: 'identity.xsl',
+                        match: 'node()|@*',
+                        prio: '-0.5',
+                        matches: 'N u[NT,NP,NC,NE]',
+                        C: [
+                          {
+                            N: 'p.nodeTest',
+                            role: 'match',
+                            test: 'N u[NT,NP,NC,NE]',
+                            sType: '1N u[NT,NP,NC,NE]',
+                          },
+                          {
+                            N: 'copy',
+                            sType: '1N u[1NT ,1NP ,1NC ,1NE ] ',
+                            flags: 'cin',
+                            role: 'action',
+                            line: '4',
+                            C: [
+                              {
+                                N: 'applyT',
+                                sType: '* ',
+                                line: '5',
+                                mode: '#unnamed',
+                                bSlot: '0',
+                                C: [
+                                  {
+                                    N: 'docOrder',
+                                    sType:
+                                      '*N u[N u[N u[N u[NT,NP],NC],NE],NA]',
+                                    role: 'select',
+                                    line: '5',
+                                    C: [
+                                      {
+                                        N: 'union',
+                                        op: '|',
+                                        sType:
+                                          '*N u[N u[N u[N u[NT,NP],NC],NE],NA]',
+                                        ns: '= xml=~ fn=~ xsl=~ ',
+                                        C: [
+                                          {
+                                            N: 'axis',
+                                            name: 'child',
+                                            nodeTest: '*N u[NT,NP,NC,NE]',
+                                          },
+                                          {
+                                            N: 'axis',
+                                            name: 'attribute',
+                                            nodeTest: '*NA',
+                                          },
+                                        ],
+                                      },
+                                    ],
+                                  },
+                                ],
+                              },
+                            ],
+                          },
+                        ],
+                      },
+                      {
```

Comment:
NB: What is the purpose of the XSLT identity transform here. Do the use of absolute file paths from your workstation's file structure cause issues?

Reply:
It is used to indent the XML. The file path is not used, but can't be removed due to a checksum validation on the object. If we continue down this path, we might want to indent using another method, but this was quick and didn't require adding another dependency.

Reply:
OK, I put NB because it was largely curiosity. No biggie!